src/main/java/htsjdk/samtools/cram/encoding/core/huffmanUtils/HuffmanBitCode.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  * ****************************************************************************
  */
-package htsjdk.samtools.cram.encoding.huffman.codec;
```

Comment:
Grouped the Huffman Codecs/Encodings with the other Codecs/Encodings, and moved Huffman-specific utils to a new sub-package